CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+# Steps to contribute to the repo
+Please follow our guidlines if you wish to contribute to this repo.
+
+1. Create an issue detailing the changes you wish to make. This can be as simple as describing the file(s) you want to add, or the changes you wish to make to file(s).
+	1. Descriptions aren't needed in simple issues, but more documentation is always prefered and appriciated.
+2. Fork the repo. In github (or the git version control system of your choice) fork this repo to your own profile.
+3. Create a new branch on your forked repo named after the issue. Never make changes directly to the main branch.
+	1. For example, if you created issue #8 in the original repo, then name your branch in your fork something along the lines of "issue-08".
+	2. Commit all of your changes to his new branch in your forked repo.
+	3. Make sure to add your name to the AUTHORS.md file so you can get credit for your work!
+4. Submit a pull request for your branch into the development branch. 
+	1. This pull request will be reviewed and needs to be accepted by at least one reviewer. 
+	2. If you would like to be a reviewer, please contact one of the authors listed in the AUTHORS.md file.
```

Comment:
Tell them to contact me instead, since I'm the one who can add contributors.
Also tell them to message me on Teams if they don't hear back on GitHub.